src/Microsoft.SqlTools.ServiceLayer/Connection/Contracts/GetConnectionStringParams.cs

```diff
@@ -19,5 +19,10 @@ public class GetConnectionStringParams
         /// Indicates whether the password should be return in the connection string
         /// </summary>
         public bool IncludePassword { get; set; }
+
+        /// <summary>
+        /// Indicates whether the application name should be return in the connection string
+        /// </summary>
+        public bool IncludeApplicationName{ get; set; }
```

Comment:
I'm not at all familiar with STS code, so just asking questions 😄.

Are all of these params optional, or is this now a required field? Do we need a `?`?

Also nit: spacing (`IncludeApplicationName {` instead of `IncludeApplicationName{`)

Reply:
I followed the same pattern as above since I am not that familiar as well. All these parameters are optional, we make ownerURI required in the many different places we utilize getConnectionStringRequest as that is what we use to set build the connectionString in STS.

Reply:
I imagine for this you're going to want to make it optional. I haven't looked too closely at the code that turns the JSON params into objects but my guess is that it just uses the default for the type (false in this case) if it isn't specified when deserializing into the object. Hopefully whatever does that would then leave that value as null instead if it wasn't specified.